frontend/lib/components/TextEditor/TextEditor.module.scss

```diff
@@ -27,11 +28,27 @@
 .editor_wrapper {
   min-height: 100vh;
 
+  .bubble_menu {
+    background-color: var(--background-3);
+    border: solid 1px var(--border-0);
+    border-radius: Radius.$normal;
+  }
+
   .content_wrapper {
     color: var(--text-3);
     width: 100%;
     position: relative;
 
+    .ai_highlight {
+      background-color: blue;
+      color: white;
+      padding: 2px 0 3px 0;
+    }
```

Comment:
not sure about this ? 
Could you use Spacings variables, and use a more standard color like background-3 or somehting like this ? :)